static/js/containers/CourseIndexPage.js

```diff
@@ -123,7 +122,7 @@ const mapStateToProps = (state: Object): StateProps => ({
 })
 
 const mapDispatchToProps = {
-  setShowCourseDrawer
+  setShowResourceDrawer: setShowResourceDrawer
```

Comment:
if you want to you can leave out the colon here, it's called a 'shorthand property name' https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015